fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/SimpleSchemaPrinter.java

```diff
@@ -6,32 +6,3565 @@
+        public void updateBlob(int columnIndex, InputStream inputStream, long length) throws SQLException {
+
+        }
+
+        @Override
+        public void updateBlob(String columnLabel, InputStream inputStream, long length) throws SQLException {
+
+        }
+
+        @Override
+        public void updateClob(int columnIndex, Reader reader, long length) throws SQLException {
+
+        }
+
+        @Override
+        public void updateClob(String columnLabel, Reader reader, long length) throws SQLException {
+
+        }
+
+        @Override
+        public void updateNClob(int columnIndex, Reader reader, long length) throws SQLException {
+
+        }
+
+        @Override
+        public void updateNClob(String columnLabel, Reader reader, long length) throws SQLException {
+
+        }
+
+        @Override
+        public void updateNCharacterStream(int columnIndex, Reader x) throws SQLException {
+
+        }
+
+        @Override
+        public void updateNCharacterStream(String columnLabel, Reader reader) throws SQLException {
+
+        }
+
+        @Override
+        public void updateAsciiStream(int columnIndex, InputStream x) throws SQLException {
+
+        }
+
+        @Override
+        public void updateBinaryStream(int columnIndex, InputStream x) throws SQLException {
+
+        }
+
+        @Override
+        public void updateCharacterStream(int columnIndex, Reader x) throws SQLException {
+
+        }
+
+        @Override
+        public void updateAsciiStream(String columnLabel, InputStream x) throws SQLException {
+
+        }
+
+        @Override
+        public void updateBinaryStream(String columnLabel, InputStream x) throws SQLException {
+
+        }
+
+        @Override
+        public void updateCharacterStream(String columnLabel, Reader reader) throws SQLException {
+
+        }
+
+        @Override
+        public void updateBlob(int columnIndex, InputStream inputStream) throws SQLException {
+
+        }
+
+        @Override
+        public void updateBlob(String columnLabel, InputStream inputStream) throws SQLException {
+
+        }
+
+        @Override
+        public void updateClob(int columnIndex, Reader reader) throws SQLException {
+
+        }
+
+        @Override
+        public void updateClob(String columnLabel, Reader reader) throws SQLException {
+
+        }
+
+        @Override
+        public void updateNClob(int columnIndex, Reader reader) throws SQLException {
+
+        }
+
+        @Override
+        public void updateNClob(String columnLabel, Reader reader) throws SQLException {
+
+        }
+
+        @Override
+        public <T> T getObject(int columnIndex, Class<T> type) throws SQLException {
+
+            return null;
+        }
+
+        @Override
+        public <T> T getObject(String columnLabel, Class<T> type) throws SQLException {
+
+            return null;
+        }
+
+    }
+
+    public class PrintCallableStatement implements java.sql.CallableStatement {
```

Comment:
Is it really needed?

Reply:
I wanted to be able to cover future changes and capture them as part of the print. 
There are some db2 commands which can be executed as callables.